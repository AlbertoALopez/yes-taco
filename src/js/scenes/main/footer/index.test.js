import React from 'react';
import Footer from './index.js';
import renderer from 'react-test-renderer';

test('Footer renders correctly', () => {
  const wrapper = shallow(<Footer />);

  expect(wrapper).toMatchSnapshot();
});
