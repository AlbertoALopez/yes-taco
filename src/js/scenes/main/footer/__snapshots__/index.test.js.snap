// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="footer-text-container"
>
  <p>
    Credit to Jing-Yang for 
    <a
      href="https://itunes.apple.com/us/app/not-hotdog/id1212457521?mt=8"
      rel="noopener noreferrer"
      target="_blank"
    >
      NotHotDog
    </a>
  </p>
</div>
`;
